src/currencyEngineETH.js

```diff
@@ -889,6 +896,54 @@ class EthereumEngine {
       this.log('Error fetching networkFees:')
       this.log(err)
     }
+
+    try {
```

Comment:
every time you loop through "checkUpdateNetworkFees"  you are trying to get the fees first from the info servers, and then from ethgasstation.
The problem is that if we have a result from "ethgasstation", but then the next request fails but the request to the infoServer succeeds, we would override the fees with the infoServer ones which most likely be worse...
I would reverse the order and if "ethgasstation" works, don't even try and get the fees from the infoServer

Reply:
Yes except the infoServer is where we get the standardFeeLowAmount and standardFeeHighAmount. So long as we got these values once then we're ok. But if the first time we startup, we only get ethgasstation values, then we're incomplete. I guess I can hard code some defaults into the settings.

Reply:
Or maybe get the stuff from "ethgasstation" and if it's successful, then only get the "standardFeeLowAmount" and "standardFeeHighAmount" from the infoServer

Reply:
I coded up something to fix this only to realize I don't like it. If the ethgasstation query fails, and we don't let the info1 server fees overwrite them, then there's a chance that we'd be using the built in default values hard coded in the app. Those are the worst values we could use as they can't even be updated. So, I would claim that we're better off with the code as is.